examples/02-tips-n-tricks/06-convert-apdl-to-pymapdl.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (C) 2016 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+.. _ref_converting_apdl_to_pymapdl:
+
+==================================
+Converting APDL scripts to PyMAPDL
+==================================
+
+There two methods to convert APDL scripts to PyMAPDL ones in PyMAPDL,
+
+* `convert_apdl_block()`
+* `convert_script()`
+
+which work with strings and files respectively.
+
+This example considers a modified version of the APDL verification example 45
+as the source to be converted.
+"""
+
+import tempfile
+
+from ansys.mapdl.core import convert_apdl_block, convert_script, launch_mapdl
+
+# start MAPDL
+mapdl = launch_mapdl()
```

Comment:
I dont think you need to launch MAPDL to do the conversion.

Reply:
Done!